libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -308,6 +308,16 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
     }
 }
 
+.gd-header-countdown {
```

Comment:
Should we rename `gd-header-countdown` → `gd-header-expiration-date` to be somewhat clearer?